docs/read_the_docs/source/upgrade_scenario_config.rst

```diff
@@ -49,6 +49,14 @@ Or say we want to apply the upgrade only to houses with 3 car garages that aren'
 .. todo::
    
    Come up with some better examples here.
+   
+Currently, you can enter up to 25 options per upgrade. To allow additional options per upgrade, you would need to increase the value returned by the following method defined in ``resources/HPXMLtoOpenStudio/resources/constants.rb``:
```

Comment:
Good idea, but I think they will also have to update measure.xmls (and the outputs section of the PAT projects)? Maybe language describing how to do this should go in the advanced tutorial section.